cosmwasm/deployment/xpla/tools/package-lock.json

```diff
@@ -1,16 +1,15 @@
 {
-  "name": "tools",
-  "version": "1.0.0",
+  "name": "@wormhole-foundation/tools-cosmwasm",
```

Comment:
no longer matches the package name but 🤷 will get fixed on next update

Reply:
updated this